/// Files: fs/fuse/fuse_i.h
/// Fix: 775c5033a0d164622d9d10dd0f0a5531639ed3ed
/// Fixes: 5d069dbe8aaf2a197142558b6fb2978189ba3454

virtual detect

@err exists@
identifier inode;
position p;
@@

fuse_make_bad(struct inode *inode)
{
	... when != remove_inode_hash(inode);
*	set_bit@p(FUSE_I_BAD, &get_fuse_inode(inode)->state);
}

@script:python depends on detect@
p << err.p;
@@

coccilib.report.print_report(p[0], 'ERROR: CVE-2021-28950')
